types/react-google-recaptcha/react-google-recaptcha-tests.tsx

```diff
@@ -10,11 +10,14 @@ const invisibleRecaptcha: React.SFC = () => {
             ref={recaptchaRef}
             sitekey="xxx"
             size="invisible"
+            asyncScriptOnLoad={() => {
+                console.log('test');
```

Comment:
Why have you added a console log here?

Reply:
I added the asyncScriptOnLoad property, I added it there to test it.

Reply:
Is there more to come for this change then or is this it?

Reply:
No there’s nothing else that’s needed. This is the only thing that’s needed to fix the issue.